extensions/aproc/proc/ingest/drivers/impl/digitalglobe.py

```diff
@@ -110,6 +112,9 @@ def to_item(self, url: str, assets: list[Asset]) -> Item:
                     centroid_geom = component_geometry.Centroid()
                     centroid_geom_list = str(centroid_geom).replace("(", "").replace(")", "").split(" ")
                     centroid = [float(centroid_geom_list[1]), float(centroid_geom_list[2])]
+
+                    if order_shape_file != os.path.join(d, "GIS_FILES", file):
+                        os.remove(order_shape_file)
```

Comment:
The system should never delete any file. If it is really needed, then this must be documented and we must use the comment "!DELETE!".

https://docs.google.com/document/d/1UTm5JUZ8F9ptlcya9v88hzg-iPWxeuCCziAD_oycloo/edit?tab=t.0